### DISCUSSION TITLE
Refine handle copy UI

### DISCUSSION THREAD

**Discussion by mattstein** [craftcms/cms#10732] [Ideas] [answered]
The hover-to-copy handles in Craft 4’s tables are a convenient addition, and they could use a touch of visual refinement.

The hover animation on the widest handle needlessly shifts the rest of the table:

![example-one](https://user-images.githubusercontent.com/2488775/157754180-1766a5bd-3957-49ae-944a-225706c27226.gif)

Every new row is another thing that’ll get wiggly when the hover happens:

![example-two](https://user-images.githubusercontent.com/2488775/157754447-6fdd2ea4-9693-4117-8cec-ea78c885ca26.gif)

This feels unnecessary and slightly disorienting, and it is not one of those times where the sassy, unexpected movement translates to delight.

While we’re here, I also secretly wish that the tag’s text and icon were optically centered within its little box. Currently, the text label feels like it’s drifted upward and the icon hangs a little low:

<img src="https://user-images.githubusercontent.com/2488775/157754868-7f7a7bda-6f2f-4275-9d6c-7df7bc7845c7.png" width="450" />

## tl;dr

- [ ] don’t let the hover animation shift the table
- [ ] visually center the bits and pieces in the box

**Comment by brandonkelly** [comment, marked as answer]
Resolved for the next release! (6f5d4df5eebe733ecb8b7bb4eb7147d0198cc31c)
